astrosat_users/views/views_auth.py

```diff
@@ -226,8 +227,21 @@ class PasswordResetConfirmView(RestAuthPasswordResetConfirmView):
     ```
     """
 
-    pass
+    def post(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
 
+        serializer.is_valid(raise_exception=True)
```

Comment:
What does the `raise_exception` do? I don't expect it to raise an exception, if the serializer is valid.

Reply:
It raises an exception if the serializer is invalid.  (Otherwise, `is_valid` would just return False and I would have to handle it myself).